### PR TITLE
Fix type mismatch

### LINKED ISSUE #68: window.rs:327:38: 327:67 error: mismatched types: expected `*const std::os::raw::c_void`, found `*const libc::c_void`
When attempting to build the readme sample (and installing opengl `freeglut3-dev`, and randr `xorg-dev libglu1-mesa-dev`) I'm getting this:

``` text
Executing: cargo build 
   Compiling kiss3d v0.1.4 (https://github.com/sebcrozet/kiss3d#98f00870)
~/.cargo/git/checkouts/kiss3d-fdfcc5cc23d2dd9f/master/src/window/window.rs:327:38: 327:67 error: mismatched types:
 expected `*const std::os::raw::c_void`,
    found `*const libc::c_void`
(expected enum `std::os::raw::c_void`,
    found enum `libc::c_void`) [E0308]
~/.cargo/git/checkouts/kiss3d-fdfcc5cc23d2dd9f/master/src/window/window.rs:327         verify!(gl::load_with(|name| window.get_proc_address(name)));
                                                                                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~/.cargo/git/checkouts/kiss3d-fdfcc5cc23d2dd9f/master/src/window/window.rs:327:9: 327:70 note: in this expansion of verify! (defined in ~/.cargo/git/checkouts/kiss3d-fdfcc5cc23d2dd9f/master/src/error.rs)
~/.cargo/git/checkouts/kiss3d-fdfcc5cc23d2dd9f/master/src/window/window.rs:327:38: 327:67 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to previous error
Could not compile `kiss3d`.
```

The readme says that the latest nightlies of rust are required, but that was edited a long time ago so is that still necessary?

### PR DESCRIPTION
**Pull request by Ticki**
Fixes #68

### CODE REVIEW

File: src/window/window.rs [hunk 1]
```diff
@@ -324,7 +324,7 @@ impl Window {
 
         window.make_current();
 
-        verify!(gl::load_with(|name| window.get_proc_address(name)));
+        verify!(gl::load_with(|name| mem::transmute(window.get_proc_address(name))));
```
Comment (bjz) [inline src/window/window.rs]: You don't need a transmute here. `window.get_proc_address(name) as *const _` should suffice.

**Comment by svenstaro** [comment]
Yo what's up here?

**Comment by Ticki** [comment]
it is fixed now. See https://github.com/sebcrozet/kiss3d/issues/68.

**Comment by sebcrozet** [comment]
Yes, I've just applied the fix (which does not use `mem::transmute`). It should be available on crates.io as well. Sorry for the (long) delay.

**Pull request closed without merging**